content/docs/tech-resources/using-proxy-https-tracking-domain.md

```diff
@@ -56,18 +56,20 @@ On a Debian distribution, this command will install nginx with a sample configur
 
 Note: you must store `spgo.io` in a variable so that nginx re-resolves the domain when its TTL expires. You also have to include the `resolver` directive to explicitly specify a DNS server to resolve the hostname. By including the `valid` parameter to the directive, you can tell nginx to ignore the TTL and to re‑resolve names at a specified frequency. In the sample below, nginx re‑resolves names every 10 seconds.
 
+Note: as shown in the sample configuration file below, you should forward the `Host` header so that SparkPost can determine the tracking domain used in a request.
```

Comment:
Do we want to specify that both `X-Forwarded-Host` and `Host` headers are valid, or should `X-Forwarded-Host` be just a workaround for the CloudFront configuration?

Reply:
I can mention `X-Forwarded-Host` here if it's helpful. What do you think?